src/Cli.Tests/InitTests.cs

```diff
@@ -302,6 +302,7 @@ public void EnsureFailureOnReInitializingExistingConfig()
         [DataRow("AppService", null, null, DisplayName = "AppService with no audience and no issuer specified.")]
         [DataRow("Simulator", null, null, DisplayName = "Simulator with no audience and no issuer specified.")]
         [DataRow("AzureAD", "aud-xxx", "issuer-xxx", DisplayName = "AzureAD with both audience and issuer specified.")]
+        [DataRow("EntraId", "aud-xxx", "issuer-xxx", DisplayName = "EntraId with both audience and issuer specified.")]
```

Comment:
I see that this test is failing and I think it is because of a misspelling in `EntraId`, if you go to `SupportedAuthNProviders.cs` you can see that it is spelled as `EntraID`.

```suggestion
        [DataRow("EntraID", "aud-xxx", "issuer-xxx", DisplayName = "EntraID with both audience and issuer specified.")]
```